app/views/my/_sort_and_per_page.html.erb

```diff
@@ -18,16 +18,20 @@
   <div class="sort-toggle">
     <% unless @response.response['numFound'] < 2 %>
       <%= form_tag sufia.dashboard_files_path, method: :get, class: 'per_page form-inline' do %>
-        <%= label_tag(:sort, "<span>Sort By:</span>".html_safe) %>
-        <%= select_tag(:sort, options_for_select(sort_fields, h(params[:sort]))) %>			      
-        &nbsp;&nbsp;&nbsp;
-        <%= label_tag :per_page do %>
-          Show <%= select_tag :per_page, options_for_select(['10', '20', '50', '100'], h(params[:per_page])), 
-                              title: "Number of results to display per page" %> per page
-        <% end %>
-        <%= render_hash_as_hidden_fields params_for_search().except(:per_page, :sort, :utf8) %>
-        &nbsp;&nbsp;&nbsp;
-        <button class="btn btn-info" id="dashboard_sort_submit"><i class="glyphicon glyphicon-refresh"></i> Refresh</button>
+            <fieldset class="col-xs-12">
+              <legend class="sr-only"><%= t('sufia.sort_label') %></legend>
+              <%= label_tag(:sort, "<span>Sort By:</span>".html_safe) %>
+              <%= select_tag(:sort, options_for_select(sort_fields, h(params[:sort]))) %>
+              <%= label_tag :per_page do %>
+                  Show <%= select_tag :per_page, options_for_select(['10', '20', '50', '100'], h(params[:per_page])),
+                                      title: "Number of results to display per page" %> per page
+              <% end %>
+              <%= render_hash_as_hidden_fields params_for_search().except(:per_page, :sort, :utf8) %>
```

Comment:
Can you reformat this line per line 24 of the collections view partial above?  So like:

```ruby
<%= render_hash_as_hidden_fields(params_for_search.except(:per_page, :sort, :utf8)) %>
```